ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -50,7 +50,7 @@ type Props = OwnProps &
   ReturnType<typeof mapDispatchToProps> &
   ReturnType<typeof mapStateToProps>;
 
-const copyToClipboardWithFeedback = (text: string) => {
+export const copyToClipboardWithFeedback = (text: string) => {
```

Comment:
this should be moved under `utils` now, since it's shared across many components

Reply:
Already done in https://github.com/teamdigitale/io-app/pull/981

Reply:
May waiting for #981 merge be good before doing any integration?

Reply:
changed.